easyaccess/config_ea.py

```diff
@@ -65,6 +65,12 @@ def get_config(configfile):
                                                                                          '1000')
     if not config.has_option('easyaccess', 'autocommit'): configwrite = True;config.set('easyaccess', 'autocommit',
                                                                                         'yes')
+    if not config.has_option('easyaccess', 'trim_whitespace'): configwrite = True;config.set('easyaccess',
+                                                                                             'trim_whitespace',
+                                                                                             'yes')
+    if not config.has_option('easyaccess', 'desdm_coldefs'): configwrite = True;config.set('easyaccess',
+                                                                                           'desdm_coldefs',
```

Comment:
Where does the documentation go? In the "configcomment"? Would it be possible to more directly associate the documentation with the option defaults?